yaml/xpu_functions.yaml

```diff
@@ -341,6 +341,21 @@ supported:
   - nll_loss_forward
   - nll_loss_backward.grad_input
   - nll_loss_backward
+  - batch_norm_stats
+  - batch_norm_elemt
+  - batch_norm_elemt.out
+  - batch_norm_backward_reduce
+  - batch_norm_backward_elemt
+  - batch_norm_update_stats
+  # - native_batch_norm
```

Comment:
Why these are commented out? (if the PR is not ready for review, can you move it to draft, please?)

Reply:
I currently don't have a good approach to use CI in the draft stage. Since I can't reproduce the CI issues locally, I need to continuously trigger CI to troubleshoot.